openstudiocore/src/utilities/filetypes/EpwFile.cpp

```diff
@@ -132,67 +132,119 @@ EpwDataPoint::EpwDataPoint(int year,int month,int day,int hour,int minute,
   setLiquidPrecipitationQuantity(liquidPrecipitationQuantity);
 }
 
-boost::optional<EpwDataPoint> EpwDataPoint::fromEpwString(std::string line)
+boost::optional<EpwDataPoint> EpwDataPoint::fromEpwString(const std::string &line)
+{
+  std::vector<std::string> list;
+  boost::split(list, line, boost::is_any_of(","));

```

Comment:
Actually, @kbenne and @axelstudios we had at one point said we prefer QString to boost for string stuff.  However, with the desire to make a Qt-less utility and model target, I wonder if we should go back to boost?  Boost is at least statically linked (if not header only) and likely easier to emscripten....